crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -129,18 +129,18 @@ impl TextureAtlas {
     ///
     /// * `rect` - The section of the atlas that contains the texture to be added,
     /// from the top-left corner of the texture to the bottom-right corner
-    pub fn add_texture(&mut self, rect: Rect) -> usize {
-        self.textures.push(rect);
-        self.textures.len() - 1
+    pub fn add_sprite(&mut self, rect: Rect) -> usize {
+        self.sprites.push(rect);
+        self.sprites.len() - 1
     }
 
-    /// How many textures are in the `TextureAtlas`
+    /// How many sprites are in the `TextureAtlas`
```

Comment:
```suggestion
    /// How many sprites are in the [`TextureAtlas`]
```

While we're here, might as well linkify things.